database/migrations/2020_05_29_092823_create_locations_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLocationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('locations', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('slum_name');
+            $table->string('district');
+            $table->string('division');
```

Comment:
How about having these in their own table since there will be duplication?
We could have the locations table with the details of the different locations and then we have the the slums table with the slum names and their location id.

Reply:
OK.  This is noted.